xmake/plugins/project/vstudio/impl/vs201x.lua

```diff
@@ -335,6 +335,7 @@ function _make_targetinfo(mode, arch, target, vcxprojdir)
         end
     end
     for k, v in pairs(addrunenvs) do
+        table.sort(v)
```

Comment:
We should not sort runenvs, It may affect the order in which packages so/dll are loaded, and if there are multiple so libraries with the same name, they behave differently.

Reply:
I agree this could be an issue and same as above: I sorted it because order changed between runs, a better fix may be required here.

Reply:
I think I fixed that in target:pkgenvs(), packages order wasn't stable (because it came from a hashmap) so I made it into an array and sorted it.